examples/v1/taskruns/alpha/stepaction-git-resolver.yaml

```diff
@@ -0,0 +1,18 @@
+# TODO(#7325): use StepAction from Catalog
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  generateName: step-action-run-
+spec:
+  TaskSpec:
+    steps:
+      - name: action-runner
+        ref:
+          resolver: git
+          params:
+            - name: url
+              value: https://github.com/chitrangpatel/repo1M.git
```

Comment:
Just calling it out. You've probably already captured this in a issue. 
We should probably add a quick hello-world `StepAction` in the `Catalog` and reference it from there.

Reply:
Yes, totally agree. :D

Reply:
I will work on that this week